### ISSUE TITLE
graph/flow: conversion from char (int64) to string yields a string of one rune, not a string of digits [Go 1.15]

### ISSUE THREAD

**Issue by eclipseo** [gonum/gonum#1419]
# What are you trying to do?
Packaging gonum

### What did you do?
Run tests

### What did you expect to happen?
Tests pass.

### What actually happened?
```
# gonum.org/v1/gonum/graph/flow
./control_flow_test.go:132:40: conversion from char (int64) to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
FAIL	gonum.org/v1/gonum/graph/flow [build failed]
```

### What version of Go and Gonum are you using?
Go 1.15 rc 1 with Gonum 0.7.0

### Does this issue reproduce with the current master?
Most likely.

See https://github.com/golang/go/issues/32479

**Comment by eclipseo** [comment]
Fixed in master.

**Comment by kortschak** [comment]
FTR go1.15 is not yet officially supported with a Gonum release, it will be shortly after it has been released with Gonum version v0.8.
